L1Trigger/Phase2L1ParticleFlow/src/egamma/pftkegalgo_ref.cpp

```diff
@@ -47,8 +51,25 @@ l1ct::PFTkEGAlgoEmuConfig::IsoParameters::IsoParameters(const edm::ParameterSet
                     pset.getParameter<double>("dRMin"),
                     pset.getParameter<double>("dRMax")) {}
 
+l1ct::PFTkEGAlgoEmuConfig::CompIDParameters::CompIDParameters(const edm::ParameterSet &pset)
+    : CompIDParameters(pset.getParameter<double>("loose_wp"),
+                       pset.getParameter<double>("tight_wp"),
+                       pset.getParameter<std::string>("model")) {}
+
 #endif
 
+PFTkEGAlgoEmulator::PFTkEGAlgoEmulator(const PFTkEGAlgoEmuConfig &config)
+    : cfg(config), composite_bdt_(nullptr), debug_(cfg.debug) {
+  if (cfg.doCompositeTkEle) {
+#ifdef CMSSW_GIT_HASH
+    auto resolvedFileName = edm::FileInPath(cfg.compIDparams.conifer_model).fullPath();
+#else
+    auto resolvedFileName = cfg.compIDparams.conifer_model;
+#endif
```

Comment:
Similarly, why are there different compilation options here for CMSSW or not CMSSW?

Reply:
ditto